### DISCUSSION TITLE
editing scheduled transactions

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#66] [Q&A]
Is there a way to change the amount in a scheduled transaction after it has appeared on the account's register?  I saw that my Electric bill showed up as a pending (upcoming) transaction in my checking account register, and went to check the amount I owe.  The amount was different, so I went to adjust the scheduled transaction.  But when I come back to the register, it still shows up with the original payment.

And I can't see any way to edit the transaction in the register view, so do I just have to wait until it becomes available before I can change the amount it's displaying?

**Comment by bpaulien** [comment]
Ok, so maybe this is a bug in a different way.  I just went back into scheduled transactions, and changed the price, and saved it, and it showed up the new value in both scheduled tab, and account's register.

What I had done before was to change the amount of the scheduled transaction, and then selected the "Edit as rule" to make sure I had set up the note correctly.  Then I clicked "save", then "save" again, and the amount hadn't changed.

**Reply by MiriamNZ** [reply]
I think you have to tell it to stop being a schedule and become part of the account register. 
Click or press-hold and choose 'post transaction' (i am looking on phone app, no computer near, and cant remember what it looks like on the computer). 
It stops being grey once its posted. 
This unhooks it from the schedule and you can change the amount and its just this transaction. 
(Pretty sure; but still learning).
